src/pages/Homepage.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+const wrapperMain = {
+  paddingTop: "120px",
+};
+
+const Homepage = () => {
+  return (
```

Comment:
perhaps the return is not needed here and the syntax should be `const Homepage = () => ()`